### PR TITLE
Add helm chart

### PR DESCRIPTION
**Pull request by voron**
SSIA

### CODE REVIEW

File: charts/blockchain-etl-streaming/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+                  key: GCS_PREFIX
+            - name: GOOGLE_APPLICATION_CREDENTIALS
+              value: /var/secrets/google/key.json
+          lifecycle:
+            postStart:
+              exec:
+                command:
+                  - /bin/sh
+                  - -c
+                  - "gcloud auth activate-service-account --key-file ${GOOGLE_APPLICATION_CREDENTIALS}"
```
Comment (medvedev1088) [inline charts/blockchain-etl-streaming/templates/deployment.yaml:167]: Why is this needed?

**Review by medvedev1088 — CHANGES_REQUESTED** [review]
1. To migrate default config values from the overlay folder to helm values, I suggest creating default_values directory with respective subfolders similar to overlays structure. Then in README add a note that when installing the chart, the default values can be used and overridden with a custom values file, e.g. 

> helm install . -f default_values/bitcoin/dash/values.yaml  -f prod/bitcoin/dash/values.yaml.

2. Remove unused files - everything related to customize. If we need it for some reason we can check it out from git.

3. Update README with instructions for deploying using Helm.

File: charts/blockchain-etl-streaming/templates/deployment.yaml (hunk 1 above)
Comment (voron) [reply charts/blockchain-etl-streaming/templates/deployment.yaml:167]: > Why is this needed?

gsutil [requires this](https://cloud.google.com/storage/docs/authentication#gsutilauth).

**Comment by voron** [comment]
> 1. To migrate default config values from the overlay folder to helm values, I suggest creating default_values directory with respective subfolders similar to overlays structure.

What do you think regarding complete copy of example values, with some values layering when required(ethereum as example)? All the values files contain uniq data only.

**Comment by voron** [comment]
2 and 3 are implemented.
